File: Desktop/ailydian-ultra-pro/public/docs/sdks/csharp/Models/City.cs
using System.Collections.Generic;

namespace Lydian.SDK.Models
{
    public class City
    {
        public string? Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Country { get; set; } = string.Empty;
        public int? Population { get; set; }
        public string? Timezone { get; set; }
        public Dictionary<string, object>? Metadata { get; set; }
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
    }

    public class CityAsset
    {
        public string? Id { get; set; }
        public string CityId { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Status { get; set; } = "active";
    }

    public class Alert
    {
        public string? Id { get; set; }
        public string CityId { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty;
        public string Severity { get; set; } = string.Empty;
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string Status { get; set; } = "open";
    }
}

File: public/docs/sdks/java/src/main/java/com/lydian/sdk/models/City.java
package com.lydian.sdk.models;

import java.util.Map;

public class City {
    private String id;
    private String name;
    private String country;
    private Integer population;
    private String timezone;
    private Map<String, Object> metadata;
    private String createdAt;
    private String updatedAt;

    // Getters and Setters
    public String getId() { return id; }
    public void setId(String id) { this.id = id; }

    public String getName() { return name; }
    public void setName(String name) { this.name = name; }

    public String getCountry() { return country; }
    public void setCountry(String country) { this.country = country; }

    public Integer getPopulation() { return population; }
    public void setPopulation(Integer population) { this.population = population; }

    public String getTimezone() { return timezone; }
    public void setTimezone(String timezone) { this.timezone = timezone; }

    public Map<String, Object> getMetadata() { return metadata; }
    public void setMetadata(Map<String, Object> metadata) { this.metadata = metadata; }

    public String getCreatedAt() { return createdAt; }
    public void setCreatedAt(String createdAt) { this.createdAt = createdAt; }

    public String getUpdatedAt() { return updatedAt; }
    public void setUpdatedAt(String updatedAt) { this.updatedAt = updatedAt; }
}
